testnet/.env

```
@@ -5,17 +5,12 @@ GC_MODE=archive
 NETWORK=testnet
 PRIVATE_KEY=3333333333333333333333333333333333333333333333333333333333333333
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PRIVATE_KEY key should go before the SYNC_MODE key

(UnorderedKey)